omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestConstants.java

```diff
@@ -206,4 +206,8 @@ public class RestConstants {
 	 * Constants used for the Server Log REST Service privilege checking
 	 */
 	public static final String PRIV_GET_SERVER_LOGS = "Get Server Logs";
+	/**
+	 * Constants used for the StackTrace Details in error response
```

Comment:
`Constants` or `Constant`?

Reply:
I would also be more specific when describing the use for stack trace details? How does this constant affect the stack trace details?